.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 test/dist/
+dist/*.d.ts
```

Comment:
It is very strange to me to see a dist directory checked in like this, so I didn't want to version identical types in `src` and `dist`, but felt they belonged in `src`, so I ignored them here.

Reply:
agreed for the `dist` being checked-in. 
That's weird. 
I should use the `prepare` npm script to build the lib instead.